nemo_text_processing/text_normalization/en/taggers/measure.py

```diff
@@ -202,10 +242,29 @@ def __init__(self, cardinal: GraphFst, decimal: GraphFst, fraction: GraphFst, de
             | address
             | math
         )
+
         final_graph = self.add_tokens(final_graph)
         self.fst = final_graph.optimize()
 
-    def get_address_graph(self, cardinal):
+    def get_range(self, cardinal: GraphFst):
```

Comment:
why not using range class?

Reply:
to have strictly cardinal format, i.e. one thousand nine hundred ... not date format nineteen ...